src/server/mappings/kibana_index_mappings_mixin.js

```diff
@@ -29,6 +29,9 @@ const BASE_SAVED_OBJECT_MAPPINGS = {
   doc: {
     dynamic: 'strict',
     properties: {
+      namespace: {
```

Comment:
Now that the base mappings have a `namespace` property, we can probably remove the `spaceId` property from the Spaces [plugin mappings](https://github.com/kobelb/kibana/blob/ns/x-pack/plugins/spaces/mappings.json#L2:L4).

Reply:
...and all associated esArchiver mappings 😉